src/micrate.cr

```diff
@@ -140,7 +143,7 @@ module Micrate
   private def self.migrations_by_version(migrations_path)
     Dir.entries(migrations_path)
        .select { |name| File.file? File.join(migrations_path, name) }
-       .select { |name| /^\d+_.+\.sql$/ =~ name }
+       .select { |name| /^\d+.+\.sql$/ =~ name }
```

Comment:
Not convinced about this change, as discussed in #13. Feel free to open a separate ticket / pull request if you want to discuss if further. Thanks!